website_sale_b2x_alt_price/tests/test_ui.py

```diff
@@ -7,6 +7,16 @@
 class UICase(HttpCase):
     def setUp(self):
         super().setUp()
+        # Create and select a pricelist and reset currency conversion rates
+        # to make tests pass no matter what l10n package is enabled
+        website = self.env["website"].get_current_website()
+        pricelist = self.env["product.pricelist"].create({
+            "name": "website_sale_b2x_alt_price public",
+            "currency_id": website.user_id.company_id.currency_id.id,
+            "selectable": True,
+        })
+        website.user_id.property_product_pricelist = pricelist
```

Comment:
I think this line and the following are not needed

Reply:
Then how do you specify which pricelist should the tour start with?
Regarding the other lines, they seem to fix the build in the integration environment, so I think I'd prefer to keep them... :roll_eyes:

Reply:
Well, AFAIK, you are selecting the pricelist in the tour, but OK if you don't do for "the default". About currency rate conversion, it shouldn't be needed if you are using the same currency than the company.

Reply:
> AFAIK, you are selecting the pricelist in the tour

No, it expects to have one, then switch it to other, then back to the 1st.



> About currency rate conversion, it shouldn't be needed if you are using the same currency than the company

Ok, I'll remove them and retry downstream.

Reply:
You're right, the rate conversion stuff was not required. Ready to merge then. Thanks!